lake-manifest.json

```diff
@@ -4,7 +4,7 @@
  [{"url": "https://github.com/leanprover/std4",
    "type": "git",
    "subDir": null,
-   "rev": "16d8352f7ed0d38cbc58ace03b3429d693cf50c6",
```

Comment:
http://GitHub.com/leanprover/std4/compare/16d8352f7ed0d38cbc58ace03b3429d693cf50c6...6b4cf96c89e53cfcd73350bbcd90333a051ff4f0

Reply:
What's this?